app/src/main/java/com/breadwallet/tools/animation/BRAnimator.java

```diff
@@ -324,10 +324,11 @@ public void run() {
         } else return false;
     }
 
-    public static void killAllFragments(Activity app) {
+    public static void killAllFragments(FragmentActivity app) {
```

Comment:
nice...I'm a weirdo when it comes to naming classes..the more the verbose the better. You will see many abbr. (😎) and they making the code really hard to follow. 
Thanks for this @andhikayuana

Reply:
sure 🤝